packages/mui-joy/src/Button/Button.test.js

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { describeConformance, createRenderer } from 'test/utils';
+import Button, { buttonClasses as classes } from '@mui/joy/Button';
+import { ThemeProvider } from '@mui/joy/styles';
+
+describe('Joy <Button />', () => {
+  const { render } = createRenderer();
+
+  describeConformance(<Button>Conformance?</Button>, () => ({
+    render,
+    classes,
+    ThemeProvider,
+    refInstanceof: window.HTMLButtonElement,
+    muiName: 'MuiButton',
+    testVariantProps: { variant: 'contained', fullWidth: true },
+    testStateOverrides: { prop: 'size', value: 'small', styleKey: 'sizeSmall' },
+    skip: ['propsSpread', 'componentsProp', 'classesRoot'],
+  }));
+
+  it('by default, should render with the root, variantText, sizeMedium and colorBrand classes', () => {
```

Comment:
Based on the tests if the button is rendered with all classes all test would pass :) Shouldw e maybe test that by default the other classes are not added? Or update the other test to test that the class is not there by default, but it is if the props is added.

Reply:
> Shouldw e maybe test that by default the other classes are not added

Added.